book/analysis/introduction.md

```diff
@@ -123,18 +122,16 @@ Sum is 50000005000000, required 0.00000095 seconds
 Sum is 5000000050000000, required 0.00000119 seconds
 ```
 
-There are a couple things to notice about this output.
+There are a couple things to notice about this output:
```

Comment:
No, [you're right](https://painintheenglish.com/case/267/), I just screwed up. :wrench: